File: myblog-server/src/main/java/com/site/blog/util/UserUtil.java
package com.site.blog.util;

import com.baomidou.mybatisplus.core.conditions.query.QueryWrapper;
import com.site.blog.model.entity.AdminUser;

/**
 * @author yzq
 * @description
 * @date:Created time 2021/8/11 22:02
 * @modified By:
 */
public class UserUtil {
    public static AdminUser getUserByUserCode(String userCode) {

        AdminUser u = new AdminUser();
        u.setId(userCode);

        return u.selectOne(new QueryWrapper<>(u));

    }
}

File: blog-sever/server-kt/src/main/kotlin/com/site/blog/util/UserUtil.kt
package com.site.blog.util

import com.baomidou.mybatisplus.core.conditions.query.QueryWrapper
import com.site.blog.model.entity.AdminUser


object UserUtil {
    fun getUserByUserCode(userCode: String?): AdminUser? {
        val u = AdminUser()
        u.id = userCode
        return u.selectOne(QueryWrapper(u))
    }
}